src/test/resources/META-INF/services/com.redhat.vertx.pipeline.Step

```diff
@@ -0,0 +1,4 @@
+com.redhat.vertx.pipeline.step.HelloWorldStep
+com.redhat.vertx.pipeline.step.AbstractStepTest$Concat
+com.redhat.vertx.pipeline.steps.Copy
+com.redhat.vertx.pipeline.Section
```

Comment:
Not sure if this is needed, but doesn't hurt.

Reply:
2 tests failed without all 4 lines.